torch/csrc/jit/runtime/interpreter.cpp

```diff
@@ -1344,12 +1344,12 @@ struct InterpreterStateImpl : c10::intrusive_ptr_target {
   }
 
   void handleError(const ExceptionMessage& msg, bool is_jit_exception) {
-    std::stringstream ss;
+    std::ostringstream ss;
```

Comment:
Either is fine, it's a super minor micro-opt.

stringstream is effectively both an input and output stringstream. If you only need half the functionality, the constructor is cheaper for ostringstream. Because of c++ locale requirements, the constructor is a bit more expensive than you might expect